src/map/script.cpp

```diff
@@ -16497,20 +16496,28 @@ BUILDIN_FUNC(summon)
 		check_event(st, event);
 	}
 
-	clif_skill_poseffect(&sd->bl,AM_CALLHOMUN,1,sd->bl.x,sd->bl.y,tick);
+	mob_data* md = mob_once_spawn_sub( &sd->bl, sd->bl.m, sd->bl.x, sd->bl.y, str, _class, event, SZ_SMALL, AI_NONE );
+
+	if( md == nullptr ){
+		ShowError( "buildin_summon: Invalid mob ID %d.\n", _class );
+		return SCRIPT_CMD_FAILURE;
```

Comment:
A value must be returned or the script should stop (`SCRIPT_CMD_FAILURE` does not stop the script)
```suggestion
		st->state = END;
		return SCRIPT_CMD_FAILURE;
```